### PR TITLE
New CTDMO cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
These cal csvs were autogenerated during QCT testing.  I thoroughly reviewed them before posting them to this PR.  They only need one additional review.

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PYIV Instrument Cal CSVs [DT-158]](https://trello.com/c/I1InRCiO/158-pyiv-instrument-cal-csvs-dt-158)

File: calibration/CTDMOQ/CGINS-CTDMOQ-12934__20220218.csv [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+37-12934,CC_ptca0,5.240990e+05,
+37-12934,CC_ptca1,2.166621e+00,
+37-12934,CC_ptca2,1.202263e-03,
+37-12934,CC_ptcb0,2.516488e+01,
+37-12934,CC_ptcb1,-2.250000e-04,
+37-12934,CC_ptcb2,0.000000e+00,
+37-12934,CC_ptempa0,-6.698576e+01,
+37-12934,CC_ptempa1,5.303713e-02,
+37-12934,CC_ptempa2,-8.124370e-07,
+37-12934,CC_wbotc,9.659268e-08,
```
Comment (cdobs) [inline calibration/CTDMOQ/CGINS-CTDMOQ-12934__20220218.csv:25]: looks good

File: calibration/CTDMOQ/CGINS-CTDMOQ-12935__20220218.csv [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+37-12935,CC_ptca0,5.250595e+05,
+37-12935,CC_ptca1,-5.364724e+00,
+37-12935,CC_ptca2,1.387018e-01,
+37-12935,CC_ptcb0,2.503500e+01,
+37-12935,CC_ptcb1,-0.000000e+00,
+37-12935,CC_ptcb2,0.000000e+00,
+37-12935,CC_ptempa0,1.435421e+02,
+37-12935,CC_ptempa1,-6.633316e-02,
+37-12935,CC_ptempa2,4.744847e-08,
+37-12935,CC_wbotc,2.900788e-07,
```
Comment (cdobs) [inline calibration/CTDMOQ/CGINS-CTDMOQ-12935__20220218.csv:25]: looks good

File: calibration/CTDMOQ/CGINS-CTDMOQ-12936__20220218.csv [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+37-12936,CC_ptca0,5.250069e+05,
+37-12936,CC_ptca1,-4.363454e+00,
+37-12936,CC_ptca2,2.453982e-01,
+37-12936,CC_ptcb0,2.502400e+01,
+37-12936,CC_ptcb1,-0.000000e+00,
+37-12936,CC_ptcb2,0.000000e+00,
+37-12936,CC_ptempa0,1.431774e+02,
+37-12936,CC_ptempa1,-6.985056e-02,
+37-12936,CC_ptempa2,8.874242e-07,
+37-12936,CC_wbotc,5.123428e-07,
```
Comment (cdobs) [inline calibration/CTDMOQ/CGINS-CTDMOQ-12936__20220218.csv:25]: looks good

File: calibration/CTDMOQ/CGINS-CTDMOQ-12937__20220219.csv [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+37-12937,CC_ptca0,5.262229e+05,
+37-12937,CC_ptca1,3.766854e+00,
+37-12937,CC_ptca2,4.527021e-02,
+37-12937,CC_ptcb0,2.520175e+01,
+37-12937,CC_ptcb1,3.500000e-04,
+37-12937,CC_ptcb2,0.000000e+00,
+37-12937,CC_ptempa0,1.331657e+02,
+37-12937,CC_ptempa1,-6.610543e-02,
+37-12937,CC_ptempa2,1.952656e-07,
+37-12937,CC_wbotc,4.743572e-07,
```
Comment (cdobs) [inline calibration/CTDMOQ/CGINS-CTDMOQ-12937__20220219.csv:25]: looks good

File: calibration/CTDMOQ/CGINS-CTDMOQ-12938__20220219.csv [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+37-12938,CC_ptca0,5.255764e+05,
+37-12938,CC_ptca1,-8.362938e+00,
+37-12938,CC_ptca2,3.389020e-01,
+37-12938,CC_ptcb0,2.521475e+01,
+37-12938,CC_ptcb1,-2.500000e-04,
+37-12938,CC_ptcb2,0.000000e+00,
+37-12938,CC_ptempa0,1.428686e+02,
+37-12938,CC_ptempa1,-6.798780e-02,
+37-12938,CC_ptempa2,7.621434e-07,
+37-12938,CC_wbotc,2.288518e-07,
```
Comment (cdobs) [inline calibration/CTDMOQ/CGINS-CTDMOQ-12938__20220219.csv:25]: looks good

File: calibration/CTDMOQ/CGINS-CTDMOQ-12939__20220219.csv [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+37-12939,CC_ptca0,5.245251e+05,
+37-12939,CC_ptca1,3.934698e+00,
+37-12939,CC_ptca2,-5.241406e-02,
+37-12939,CC_ptcb0,2.507037e+01,
+37-12939,CC_ptcb1,2.750000e-04,
+37-12939,CC_ptcb2,0.000000e+00,
+37-12939,CC_ptempa0,1.636586e+02,
+37-12939,CC_ptempa1,-7.026830e-02,
+37-12939,CC_ptempa2,1.585617e-06,
+37-12939,CC_wbotc,6.565093e-08,
```
Comment (cdobs) [inline calibration/CTDMOQ/CGINS-CTDMOQ-12939__20220219.csv:25]: looks good

File: calibration/CTDMOQ/CGINS-CTDMOQ-13555__20220219.csv [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+37-13555,CC_ptca0,5.266451e+05,
+37-13555,CC_ptca1,-2.695334e+00,
+37-13555,CC_ptca2,4.155281e-01,
+37-13555,CC_ptcb0,2.539075e+01,
+37-13555,CC_ptcb1,1.950000e-03,
+37-13555,CC_ptcb2,0.000000e+00,
+37-13555,CC_ptempa0,1.526184e+02,
+37-13555,CC_ptempa1,-6.461878e-02,
+37-13555,CC_ptempa2,-5.454330e-07,
+37-13555,CC_wbotc,-2.801068e-07,
```
Comment (cdobs) [inline calibration/CTDMOQ/CGINS-CTDMOQ-13555__20220219.csv:25]: looks good

File: calibration/CTDMOR/CGINS-CTDMOR-13628__20220219.csv [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+37-13628,CC_ptca0,5.247070e+05,
+37-13628,CC_ptca1,2.100586e-01,
+37-13628,CC_ptca2,-5.731129e-02,
+37-13628,CC_ptcb0,2.518518e+01,
+37-13628,CC_ptcb1,-3.500000e-04,
+37-13628,CC_ptcb2,0.000000e+00,
+37-13628,CC_ptempa0,2.108966e+02,
+37-13628,CC_ptempa1,-1.038559e-01,
+37-13628,CC_ptempa2,3.998212e-06,
+37-13628,CC_wbotc,3.204122e-07,
```
Comment (cdobs) [inline calibration/CTDMOR/CGINS-CTDMOR-13628__20220219.csv:25]: looks good

File: calibration/CTDMOR/CGINS-CTDMOR-13629__20220219.csv [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+37-13629,CC_ptca0,5.245231e+05,
+37-13629,CC_ptca1,-7.706948e-02,
+37-13629,CC_ptca2,-8.261511e-02,
+37-13629,CC_ptcb0,2.498152e+01,
+37-13629,CC_ptcb1,1.000000e-04,
+37-13629,CC_ptcb2,0.000000e+00,
+37-13629,CC_ptempa0,1.608884e+02,
+37-13629,CC_ptempa1,-8.885712e-02,
+37-13629,CC_ptempa2,-3.766969e-07,
+37-13629,CC_wbotc,1.571752e-07,
```
Comment (cdobs) [inline calibration/CTDMOR/CGINS-CTDMOR-13629__20220219.csv:25]: looks good

File: calibration/CTDMOR/CGINS-CTDMOR-13630__20220219.csv [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+37-13630,CC_ptca0,5.243552e+05,
+37-13630,CC_ptca1,4.845345e-01,
+37-13630,CC_ptca2,-4.333478e-02,
+37-13630,CC_ptcb0,2.506005e+01,
+37-13630,CC_ptcb1,-3.750000e-04,
+37-13630,CC_ptcb2,0.000000e+00,
+37-13630,CC_ptempa0,2.029039e+02,
+37-13630,CC_ptempa1,-9.665064e-02,
+37-13630,CC_ptempa2,1.489440e-06,
+37-13630,CC_wbotc,4.468605e-07,
```
Comment (cdobs) [inline calibration/CTDMOR/CGINS-CTDMOR-13630__20220219.csv:25]: looks good

File: calibration/CTDMOR/CGINS-CTDMOR-13654__20220302.csv [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+37-13654,CC_ptca0,5.241641e+05,
+37-13654,CC_ptca1,3.021575e-01,
+37-13654,CC_ptca2,5.048573e-03,
+37-13654,CC_ptcb0,2.520675e+01,
+37-13654,CC_ptcb1,-6.500000e-04,
+37-13654,CC_ptcb2,0.000000e+00,
+37-13654,CC_ptempa0,1.941926e+02,
+37-13654,CC_ptempa1,-8.890059e-02,
+37-13654,CC_ptempa2,-1.034340e-08,
+37-13654,CC_wbotc,0.000000e+00,
```
Comment (cdobs) [inline calibration/CTDMOR/CGINS-CTDMOR-13654__20220302.csv:25]: looks good

**Review by cdobs — APPROVED** [review]
everything looks good

**Pull request merged**